examples/util/OpenCVRunner.js

```diff
@@ -0,0 +1,106 @@
+class OpenCVRunner extends BaseRunner {
+  constructor() {
+    super();
+    this._output = null;
+  }
+
+  /**
+   * This method is to load model file with specified url.
+   * @param url: A string for model file url.
+   */
+  _loadModelFile = async (url) => {
+    if (url !== undefined) {
+      const arrayBuffer = await this._loadURL(url, this._progressHandler, true);
+      const bytes = new Uint8Array(arrayBuffer);
+      switch (url.split('.').pop()) {
+        case 'onnx':
+          const err = onnx.ModelProto.verify(bytes);
+          if (err) {
+            throw new Error(`The model file ${url} is invalid, ${err}`);
+          }
+          try {
+            cv.FS_createDataFile('/', this._currentModelInfo.modelId, bytes, true, false, false);
+          } catch (e) {
+            if (e.errno === 17) {
+              console.log(`${this._currentModelInfo.modelId} already exited.`);
+            } else {
+              console.error(e);
+            }
+          }
+          break;
+        default:
+          throw new Error(`Unrecognized model format, support TFLite | ONNX | OpenVINO model`);
+      }
+    } else {
+      throw new Error(`There's none model file info, please check config info of modelZoo.`);
+    }
+
+    this._setLoadedFlag(true);
+  };
+
+  _doInitialization = (modelInfo) => {
+    this._setLoadedFlag(false);
+    this._setInitializedFlag(false);
+    this._setModelInfo(modelInfo);
+  };
+
+  _doCompile = (options) => {
+    let model = null;
+
+    if (this._currentModelInfo.format === 'ONNX') {
+      model = cv.readNetFromONNX(this._currentModelInfo.modelId);
+      this._setModel(model);
+    } else {
+      throw new Error(`Unsupported '${this._currentModelInfo.format}' input`);
+    }
+  };
+
+  /**
+   * This method is to get input tensor with input src (HTML [<img> | <video>] element).
+   * @param src: An object for HTML [<img> | <video>] element.
+   * @param options: A string to get inputTensor. Details:
+   * options = {
+   *   // inputSize was configed in modelZoo.js, inputSize = [h, w, c].
+   *   inputSize: inputSize,
+   *   // preOptions was also configed in modelZoo.js,
+   *   // preOptions= {mean: [number, number, number, number],std: [number, number, number, number]}
+   *   preOptions: preOptions,
+   *   imageChannels: 4,
+   * };
+   * @returns {object} This returns an object for input tensor.
+   */
+  _getInputTensor = (input) => {
```

Comment:
Only keep `/** @override */` here, please move the method JS Doc into `BaseRunner`.